ted2zim/entrypoint.py

```diff
@@ -14,11 +14,29 @@ def main():
         prog=NAME, description="Scraper to create ZIM files from TED videos",
     )
 
+    parser.add_argument(
+        "--categories",
+        help="Comma-seperated list of topics to scrape. Should be exactly same as given on ted.com/talks",
+        required=True,
+    )
+
+    parser.add_argument(
+        "--max-videos",
+        help="Max number of videos to scrape in each topic. Pass 'max' if you want to scrape all",
```

Comment:
Scraping all should be the default behavior. If you want to postpone that, we can open a ticket but then the special value should be `all` rather than `max`.

Reply:
We pass a large number as default value (for now). This makes scraping all as the default behaviour.